packages/runner-shared/src/logger.js

```diff
@@ -30,7 +30,8 @@ export const logger = {
     _.each(formattedLog, (value, key) => {
       // don't log empty strings
       // _.trim([]) returns '' but we want to log empty arrays, so account for that
-      if (_.trim(value) === '' && !_.isArray(value)) return
+      // Return early if a javascript object that isn't an Array.
+      if (!_.isArray(value) && typeof value === 'object' && _.trim(value) === '') return
```

Comment:
I think my original comment was misunderstood slightly, I was saying to skip the `_.trim` call if it's an object, so Cypress would still call `this.log(`%c${key}`, 'font-weight: bold', value)` if `value` is an object. It would just not try to `_.trim` it to compare it to `''`.